x-pack/plugins/alerting/server/task_runner/create_execution_handler.ts

```diff
@@ -203,5 +206,9 @@ export function createExecutionHandler<
         alertSubgroup: actionSubgroup,
       });
     }
+
```

Comment:
We only want to call `alertingEventLogger.logAction` when we've successfully enqueued the action, so we probably want to move that out of the for loop and after the `actionsClient.bulkEnqueueExecution`. That will add another for loop unfortunately, but we don't want to be writing out the event log docs unless we're sure the actions were scheduled.

Reply:
Resolved in this [commit](https://github.com/elastic/kibana/pull/137781/commits/b15ed37052ae2e0e865df502af4255665c206408)